.eleventy.js

```diff
@@ -73,6 +73,20 @@ module.exports = (eleventyConfig) => {
   eleventyConfig.addPlugin(pluginSyntaxHighlight, {
     init({ Prism }) {
       installPrismLanguages(Prism);
+      Prism.languages.grammar = {
+        // matches the `[...]` part in `ProductionName[...]`
+        'production-params': /(?<=[a-z])\[.*?\]/,
+        // matches `ProductionName`
+        'production-name': /\b[A-Z][A-Za-z_]*\b/,
+        // "but not" and "one of" are special human-readable words in
+        // the ES spec that shouldn't be formatted as literals
+        'skip': /but not|one of/,
+        // `opt` is an ES grammar keyword to be formatted as a subscript
+        // `list+` and `list*` are Torque grammar keywords
+        'keyword': /\bopt\b|\blist[+*]/,
+        // anything else - numbers, words, punctuation - is a literal
+        'literal': /\S+/,
+      };
```

Comment:
Would it make sense to move this to its own function outside of `.eleventy.js`, similar to `installPrismLanguages`? Maybe `installPrismExtras()` which then does the two?

Reply:
Ah, I haven't noticed it's a custom module and not a 3rd-party. Looks like we already define some custom grammars like `torque` in that file, so I suppose I could just add this one to the bottom, no need to split out into more files yet.

Reply:
Done.

Reply:
Yeah, adding it to the existing file is what I was suggesting 👍 Thanks!